storage/local/crashrecovery.go

```diff
@@ -44,7 +44,9 @@ func (p *persistence) recoverFromCrash(fingerprintToSeries map[model.Fingerprint
 
 	// Delete the fingerprint mapping file as it might be stale or
 	// corrupt. We'll rebuild the mappings as we go.
-	os.Remove(p.mappingsFileName())
+	if err := os.Remove(p.mappingsFileName()); err != nil {
+		return fmt.Errorf("couldn't remove old fingerprint mapping file %s: %s", p.mappingsFileName(), err)
```

Comment:
This will return an error if the file doesn't exist. You either have to check for that, or you have to use RemoveAll. Or I don't know what...